### PR TITLE
LPS-130145 & LPS-130533 & LPS-130534 & LPS-130820 Display Pages Item Preview

### PR DESCRIPTION
**Pull request by p2kmgcl**
This PR adds pending changes we had for https://issues.liferay.com/browse/LPS-128904 to the existing PR https://github.com/liferay-echo/liferay-portal/pull/4198

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by p2kmgcl** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/App.js [hunk 1]
```diff
@@ -61,6 +72,15 @@ export default function App({state}) {
 					<DragAndDropContextProvider>
 						<EditableProcessorContextProvider>
 							<DisplayPagePreviewItemContextProvider>
+								{displayPagePreviewItemSelectorWrapper
+									? createPortal(
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/App.js:76]: Que sexy esto, mare de deu ❤️

Comment (p2kmgcl) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/App.js:76]: Me inspiré en Java para el sutil nombre `displayPagePreviewItemSelectorWrapper`

Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/App.js:76]: Me referia al portal para montar el component en la control menu

**Comment by p2kmgcl** [comment]
ci:forward

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/App.js (hunk 1 above)
Comment (p2kmgcl) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/App.js:76]: ah, tambien tambien, es precioso

Comment (p2kmgcl) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/App.js:76]: funciona™

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by p2kmgcl** [comment]
ci:test:relevant

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**